atomicapp/nulecule/container.py

```diff
@@ -160,7 +160,7 @@ def extract(self, image, source, dest, update=False):
         Utils.rm_dir(tmpdir)
 
         # Clean up dummy container
-        rm_cmd = [self.docker_cli, 'rm', '-f', container_id]
+        rm_cmd = [self.docker_cli, 'rm', '-f', container_id[:12]]
```

Comment:
Agreed with @surajssd 

In regards to this command, is it possible to have it output via the "logger.info" command?